src/media/js/apps_buttons.js

```diff
@@ -135,6 +135,12 @@ define('apps_buttons',
                 def.reject();
             });
         } else {
+            // Close the window if it was a paid app we'd logged-in for
```

Comment:
I don't think I understand this comment, could be because I'm unsure of the context here.

When install gets called we pass it a reference to a popup window if it is a paid app and if they don't require payment they already paid so we close it?